python/create-datasets-from-excel/create-datasets-from-excel.py

```diff
@@ -78,17 +130,20 @@
     dataset.metadatablocks["citation"].ds_description[0].value = description
     dataset.metadatablocks["citation"].author[0].name = agency_responsible
     dataset.metadatablocks["citation"].author[0].affiliation = None
-    dataset.metadatablocks["citation"].dataset_contact[0].name = "Harvard Dataverse Support"
-    dataset.metadatablocks["citation"].dataset_contact[0].email = "support@dataverse.harvard.edu"
-    dataset.metadatablocks["citation"].origin_of_sources = '<a href="' + access_link_url + '">' + access_link_name + '</a>'
+
+    ds_contact = dataset.metadatablocks["citation"].dataset_contact[0]
+    ds_contact.name = "Harvard Dataverse Support"
+    ds_contact.email = "support@dataverse.harvard.edu"
+
+    if access_link_url:
+        origin_of_sources = dataset.metadatablocks["citation"].origin_of_sources
+        origin_of_sources = f'<a href="{access_link_url}">{access_link_name}</a>'
 
     # Update the dataset, pushing the new fields to the server
     dataset.update()
```

Comment:
90 of the 95 datasets were created but then I got a 500 error:

- https://github.com/IQSS/dataverse/issues/11293